modules/administration_manual/pages/release_notes.adoc

```diff
@@ -1,23 +1,22 @@
 = Release Notes
 
-* xref:changes-in-10.0.10[Changes in 10.0.10]
-* xref:changes-in-10.0.9[Changes in 10.0.9]
-* xref:changes-in-10.0.8[Changes in 10.0.8]
-* xref:changes-in-10.0.7[Changes in 10.0.7]
-* xref:changes-in-10.0.6[Changes in 10.0.6]
-* xref:changes-in-10.0.5[Changes in 10.0.5]
-* xref:changes-in-10.0.4[Changes in 10.0.4]
-* xref:changes-in-10.0.3[Changes in 10.0.3]
-* xref:changes-in-10.0.1[Changes in 10.0.1]
-* xref:changes-in-10.0.0[Changes in 10.0.0]
-* xref:changes-in-9.1[Changes in 9.1]
-* xref:changes-in-9.0[Changes in 9.0]
-* xref:changes-in-8.2[Changes in 8.2]
-* xref:changes-in-8.1[Changes in 8.1]
-* xref:changes-in-8.0[Changes in 8.0]
-* xref:changes-in-7.0[Changes in 7.0]
-
-[[changes-in-10.0.10]]
+* xref:_changes_in_10_0_10[Changes in 10.0.10]
```

Comment:
The leading underscore should not exist anymore?

Reply:
Well spotted. I'll get changes made.